tests/unit/utils/Utils.h

```diff
@@ -0,0 +1,15 @@
+#ifndef __UTILS_H__
+#define __UTILS_H__
+
+/*
```

Comment:
This one is good. So you wrote this thing? Just wondering whether we should put credits for someone else?